src/brucke_config.erl

```diff
@@ -317,19 +317,25 @@ validate_ssl_files(ClientId, SslOptions) ->
   lists:foldl(
     fun(OptName, OptIn) ->
       validate_ssl_files(ClientId, OptIn, OptName)
-    end, SslOptions, [cacertfile, certfile, keyfile]).
+    end, SslOptions, [ {mandatory, cacertfile}
```

Comment:
Just for information: cacertfile should not be mandatory either, if CA is in a standard location, erlang will find it. But as of now, brod does not handle `{ssl, []}`. We should fix both eventually.